src/Features/ERDDAP/Platform/Info/index.tsx

```diff
@@ -2,7 +2,7 @@
  * Platform info panel
  */
 import { Geometry } from "@turf/helpers"
-import * as React from "react"
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)